samples/todo-app/database-api/Controllers/TodosController.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DatabaseApi.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class TodosController : ControllerBase
+    {
+        [HttpGet]
+        public IEnumerable<TodoTask> Get()
+        {
+            var todoTasks = MongoHelper.GetTasks();
+            // TODO: FIX ME BY COMMENTING THE NEXT LINE.
```

Comment:
Can we document this bug in the top-level README.md?

Reply:
Absolutely! I need to update the README for this new sample, but I don't want to make a poor text writer job. :)
I'm going to keep this PR opened and ask Gordon to write the new quickstart based on my fork, so that I can merge it later with the README updated.